src/cbdgen-framework.py

```diff
@@ -250,6 +168,7 @@ def main(seed=None):
     #dataFrame = pd.read_csv(str(N_ATTRIBUTES) + '.csv')
     #dataFrame = dataFrame.drop('c0', axis=1)
     dataFrame = df
+    ecolTest = Ecol(dataframe=dataFrame, label='label')
```

Comment:
maybe, this variable should be renamed to a more precise name, due that `Test` means in a subjective meaning.